elements/modal.js

```diff
@@ -1,6 +1,122 @@
 const widget = require('cache-element/widget')
 const Modal = require('../lib/modal-element')
 const html = require('choo/html')
+const css = require('sheetify')
+
+css('dat-colors')
+
+const prefix = css`
+  :host {
+    --input-height: 2rem;
+    --icon-height: 1.2rem;
+    --button-width: 2rem;
+    box-shadow: 0 1.2rem 2.4rem rgba(0,0,0,.5);
+    position: relative;
+  }
+  .exit {
```

Comment:
Ha yeah I was wondering the same but then noticed they get prefixed anyway, right?
See example: 
![image](https://cloud.githubusercontent.com/assets/53684/21009604/a71cb6da-bd9b-11e6-82c0-20ec20982393.png)